enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -56,6 +56,7 @@ type DockerOptions struct {
 	DockerNetwork           string
 	DockerCapAdd            string
 	DockerDevices           []config.DockerDeviceMapping
+	Binds                   []string
```

Comment:
It looks like [`Binds`](https://cs.github.com/moby/moby/blob/61404de7df1a6c75c2cbdc2c3ce226c95bebcaad/api/types/container/host_config.go#L411) specifies only volumes, while [`Mounts`](https://cs.github.com/moby/moby/blob/61404de7df1a6c75c2cbdc2c3ce226c95bebcaad/api/types/container/host_config.go#L411) can specify other types of mounts as well, including volumes.

What do you think about changing this to `Mounts` and then passing it through as `HostConfig.Mounts`?